src/content/docs/en/recipes/bun.mdx

```diff
@@ -117,6 +117,36 @@ Finally, use the following command to run your built site.
 bun ./dist/server/entry.mjs
 ```
 
+## Testing in Bun
+
+Bun ships with a fast, built-in, Jest-compatible test runner through the `bun test` command. If you like to use that read [`bun test` documentation](https://bun.sh/docs/cli/test).
+
+However, you may want to use Cypress or Playwright for a modern approach to testing web apps.
+
+### Cypress
+
+Cypress is a front-end testing tool and is on a mission to "make the testing experience enjoyable and generate developer happiness". This enables you to write end-to-end tests for your Astro site.
+
+You can install Cypress like so:
+
+```bash
+bun add cypress --dev
+```
+
+For the rest of the configuration and to start your first test, follow the rest of Cypress process in the [Astro Testing Guide](https://docs.astro.build/en/guides/testing/#configuration).
```

Comment:
This is a beautiful sentence, and an excellent use of linking. No notes. Much respect.